test/SpokePoolClient.v3Events.ts

```diff
@@ -129,6 +130,35 @@ describe("SpokePoolClient: Event Filtering", function () {
     });
   });
 
+  it("Correctly substitutes outputToken when set to 0x0", async function () {
+    const { spokePool, chainId, deploymentBlock } = originSpokePoolClient;
+    const spokePoolClient = new MockSpokePoolClient(logger, spokePool, chainId, deploymentBlock, { hubPoolClient });
+
+    const hubPoolToken = randomAddress();
+    const inputToken = randomAddress();
+    const outputToken = randomAddress();
+    hubPoolClient.setTokenMapping(hubPoolToken, originChainId, inputToken);
+    hubPoolClient.setTokenMapping(hubPoolToken, destinationChainId, outputToken);
+    hubPoolClient.setDefaultRealizedLpFeePct(toBNWei("0.0001"));
+
+    const _deposit = spokePoolClient.depositV3({
+      originChainId,
+      destinationChainId,
+      inputToken,
+      outputToken: ZERO_ADDRESS,
```

Comment:
Thoughts on adding a small comment here to highlight that this line is the main testing point?

```suggestion
      outputToken: ZERO_ADDRESS, // Set empty address 0x0 to test outputToken substitution
```

Reply:
https://github.com/across-protocol/sdk-v2/pull/603/commits/0232d53ec8bb879872c2f1505f7679273607c15d